cli/util/convert.go

```diff
@@ -41,6 +49,18 @@ func NewCommands() []cli.Command {
 					Action: sendTx,
 					Flags:  txDumpFlags,
 				},
+				{
+					Name:      "canceltx",
+					Usage:     "Cancel transaction by sending conflicting transaction",
+					UsageText: "canceltx <txid> -r <endpoint> --wallet <wallet> [--account <account>] [--wallet-config <path>]]",
+					Description: `Try to prevent transaction from entering the chain by sending more prioritized 
+conflicting transaction to the given RPC node. This command expects a transaction hash as input. 
+The conflict transaction is created automatically and signed by the given default account in wallet 
+if another account is not specified.
```

Comment:
And here we probably need to specify the behaviour of `NetworkFee` calculations for conflicting transaction. If the target transaction is in the memory pool of the provided RPC node, then `NetworkFee` value of the conflicting transaction is set to the target transaction's NetworkFee value plus one (if it's enough for conflicting transaction itself). If the target transaction is not in the memory pool, then NetworkFee calculations are being performed in a standard way based on the `calculatenetworkfee` RPC request. If the `--gas` flag is provided, then specified value will be added to the resulting conflicting transaction network fee in both cases.